dev/docker/README.md

```diff
@@ -29,10 +29,17 @@ Build the docker container "librepcb:git":
 
 Different examples to run the docker container "librepcb:git":
 
+
 ```bash
 ./run_container.sh                          # enter the bash
 ./run_container.sh qtcreator librepcb.pro   # open Qt Creator
 ./run_container.sh librepcb                 # run LibrePCB
 ./run_container.sh dia                      # run Dia (diagram editor)
 ```
 
+#### Troubleshooting
+
+* "QXcbConnection: Could not connect to display :0"
+    * Give docker the rights to access the X-Server with: `xhost +local:docker`
```

Comment:
Hmm, AFAIK this shouldn't be required if the docker container is created with the `--privileged` flag, which is the case in the `build_container.sh` script. For me it works properly when just executing the `build_container.sh` and `run_container.sh` scripts, doesn't that also work for you?

In addition, isn't the `xhost +local:docker` only valid during one X session, so after a reboot you need to execute this command again? Then it would maybe make sense to add it to the `run_container.sh` script. See also https://github.com/LibrePCB/LibrePCB/issues/132.

Reply:
it did not work for me, but this may have been because i was on an earlier version of docker - I will retry from scratch now that I am on the latest version

Reply:
Ah, I get some xauth related error when the image builds:
`xauth:  unable to link authority file /tmp/.docker.xauth, use /tmp/.docker.xauth-n`.  This is then resolvable via `xhost +local:docker`